mariadb/2.initialize-database.sql

```diff
@@ -6,8 +6,8 @@ DROP Table IF Exists received_messages;
 CREATE TABLE message (
   id VARCHAR(767) PRIMARY KEY,
   destination LONGTEXT NOT NULL,
-  headers LONGTEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
-  payload LONGTEXT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+  headers JSON NOT NULL,
```

Comment:
JSON should be supported as an **option** - I don't think it's right to make it the default.

Reply:
ok, but test cdc is more difficult, but it also should be tested with regular and json columns.
Do you have some thoughts how is better to test it?